app/assets/javascript/dropdown_select_all.js

```diff
@@ -0,0 +1,40 @@
+(function($) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  function DropdownSelectAll(options) {
+    this.$dropdowns = $(options.$el);
```

Comment:
Minor nitpick/style thing so feel free to ignore, but all these `this`s hurt my brain. Might help to assign the outer `this` to a variable so it's harder to confuse it with the ones inside the callbacks.